package.json

```diff
@@ -63,6 +63,7 @@
     ]
   },
   "devDependencies": {
+    "@types/react-outside-click-handler": "^1.3.1",
```

Comment:
Please, remove "@types/react-outside-click-handler": "^1.3.1", as you do not need it here

Reply:
ok